e2e/wdio/core/tests/multi-input.e2e-spec.ts

```diff
@@ -0,0 +1,259 @@
+import {
+    click,
+    getAttributeByName,
+    getElementArrayLength, getText, isElementDisplayed, pause,
+    refreshPage,
+    scrollIntoView, setValue,
+    waitForPresent
+} from '../../driver/wdio';
+import { MultiInputPo } from '../pages/multi-input.po';
+import {
+    fruitsArr, fruitsArr2, fruitsArr3, fruitsArrCaps, fruitsArr4, Arr
+} from '../fixtures/appData/multi-input-contents';
+describe('Multi input test suite', function() {
+    const multiInputPage = new MultiInputPo();
+    const {
+        activeDropdownButtons, activeInputs, disableInputs, options, multiInputOptions, buttonShowAll, expandedDropdown,
+        hiddenAddonButtonInputOptions, compactMultiInputOptions, multiSelectButton, approveButton, mobileInputOptions,
+        displayObjectOptions, searchTermOptions, customFilterOptions, asyncExampleOptions, reactiveFormOptions,
+        tokenOptions, templateOptions
+    } = multiInputPage;
+
+    beforeAll(() => {
+        multiInputPage.open();
+        waitForPresent(activeDropdownButtons);
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(activeDropdownButtons);
+    }, 1);
+
+    it('Check RTL/LTR orientation', () => {
+        multiInputPage.checkRtlSwitch();
+    });
+
+    it('Verify inputs should have placeholder', () => {
```

Comment:
Why are some test cases in describe blocks and others not? Maybe better to organize them all in the same pattern of describe -> it -> it -> it describe -> it -> it